travis_ci_flake8.cfg

```diff
@@ -117,3 +112,5 @@ per-file-ignores =
     src/bika/lims/interfaces/__init__.py:F401,
     src/senaite/core/exportimport/instruments/__init__.py:F401,
     src/senaite/core/behaviors/__init__.py:F401,
+    # ignore "import *" (F403, F405) in Archetypes models
+    src/bika/lims/content/*.py:F403,F405,
```

Comment:
given that these files will soon be refactored away, I thought it's not worth the effort cleaning them up

Reply:
:+1: